README.md

```diff
@@ -1 +1,67 @@
-# Awesome-Books
+[](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+> This is a site for creating a list of person's favourite books that allows to add and remove those books in and out.
+
+![screenshot](./app_screenshot.png)
+
+At this stage, it is possible to add and remove books in and out of the list.
+
+## Built With
+
+- HTML
+- JavaScript
+- GitHub
+
+## Live Demo
+
+[Click here to check how it works!](https://koomekevo.github.io/Awesome-Books/)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+If you want to see how it works without any installations, visit live demo link once again!
+
+As for the installation, follow these steps!
+
+- Open the repository
+- Download ZIP-archive to your PC
+- Unpack the archive
+- Open the index.html file
+- Use the site!
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@koomekevo](https://github.com/koomekevo)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
```

Comment:
- [ ] OPTIONAL
Kindly replace the text **Author1**  and **Author2** with your respective full name, so as to make your ReadMe more organized.

Reply:
@rloterh Thanks for the review :partying_face: We have made these changes :recycle: